pkg/apis/v2beta2/receiver_webhook.go

```diff
@@ -165,6 +169,48 @@ func (r *Receiver) validateReceiver() error {
 		}
 	}
 
+	if r.Spec.Pushover != nil {
+		// validate User Profile
+		if len(r.Spec.Pushover.Profiles) == 0 {
+			allErrs = append(allErrs, field.Required(field.NewPath("spec").Child("pushover").Child("profiles"),
+				"must be specified"))
+		} else {
+			// requirements
+			tokenRegex := regexp.MustCompile(`^[A-Za-z0-9]{30}$`)
+			deviceRegex := regexp.MustCompile(`^[A-Za-z0-9_-]{1,25}$`)
+			sounds := map[string]bool{"pushover": true, "bike": true, "bugle": true, "cashregister": true, "classical": true, "cosmic": true, "falling": true, "gamelan": true, "incoming": true, "intermission": true, "magic": true, "mechanical": true, "pianobar": true, "siren": true, "spacealarm": true, "tugboat": true, "alien": true, "climb": true, "persistent": true, "echo": true, "updown": true, "vibrate": true, "none": true}
```

Comment:
This should be defined as a const variable.

Reply:
Resolved: https://github.com/txfs19260817/notification-manager/blob/pushover/pkg/apis/v2beta2/receiver_webhook.go#L32-L34